node/prepublish.js

```diff
@@ -43,15 +43,14 @@ var repoUtils = require('./repoUtils');
 // Use npm.md as the README for the package.
 var absGitRepoPath = path.resolve(path.join(__dirname, '..', '..', '..'));
 var readmePath = path.join(absGitRepoPath, 'README.md');
-var readmeBackupPath = readmePath + '.orig';
+var readmeBackupPath = 'ORIGINAL.md';
```

Comment:
Based on npm's vague documentation on readme and some online research, yes, I think so. Only way to truly verify is once ```4.2``` goes out though.